File: Lista 6/L6_Q6.c
#include <stdio.h>

int main(){
    float v_compra;

    printf("Valor de compra: ");
    scanf("%f", &v_compra);

    if(v_compra < 10){
        v_compra += v_compra*0.7;
        printf("Valor de Venda eh R$%.2f", v_compra);
    }
    else if(v_compra < 30){
        v_compra += v_compra*0.5;
        printf("Valor de Venda eh R$%.2f", v_compra);
    }
    else if(v_compra < 50){
        v_compra += v_compra*0.4;
        printf("Valor de Venda eh R$%.2f", v_compra);
    }
        else{
        v_compra += v_compra*0.3;
        printf("Valor de Venda eh R$%.2f", v_compra);
    }

return 0;
}

File: Período_1/Lista 6/Python/L6_Q6.py
valor = float(input("\nInforme o valor de compra: "))

print()

if valor < 10:
    print(f"O valor de venda é R${valor + valor * 0.7}")
elif valor < 30:
    print(f"O valor de venda é R${valor + valor * 0.5}")
elif valor < 50:
    print(f"O valor de venda é R${valor + valor * 0.4}")
else:
    print(f"O valor de venda é R${valor + valor * 0.3}")

print()